tests/test_pythonvalidate.py

```diff
@@ -50,8 +50,6 @@ def test_basic_1(filename):
     with open(expected_filename) as fp:
         expected_data = json.load(fp)
 
-    print(expected_data)
```

Comment:
Would it be just as well squashing this into the previous commit? No worries either way.

Reply:
They are both about tests but the previous commit is a about a different test, so thought it best to leave separate